.github/labeler.yml

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Pull Request Labeler Github Action Configuration: https://github.com/marketplace/actions/labeler
+
+trunk:
+  - '**'
+INFRA:
+  - .asf.yaml
+  - .gitattributes
+  - .gitignore
+  - .github/**
+  - dev-support/**
+BUILD:
+  - any: [ '/**/pom.xml' ]
+COMMON:
+  - hadoop-common-project/**
+HDFS:
+  - hadoop-hdfs-project/**
+RBF:
+  - hadoop-hdfs-project/hadoop-hdfs-rbf/**
+HDFS-NATIVE:
```

Comment:
Show `NATIVE` name include hdfs-native and common-native(hadoop-common-project/hadoop-common/src/main/native) together?

Reply:
Done